### ISSUE TITLE
Refresh the web after booking

### ISSUE THREAD

**Issue by Wanhedaa** [Icecrown-Community/ICC-Member-Website-Public#118]
We can't get a "live refresh" for the web, but can we get a refresh after the booking?

So basically the web would be updated right after you klick on "book". Its almost a live refresh. Would be helpfull for the advertisers, who book a client at the same time with an another advertiser, because they are arguing with the GC-s about who was the first. They could check themselves after each booking, if they overbooked, or not. 

tldr:
You book a customer -> the web reloads -> you don't need to refresh manually to check if its overbook or not

**Comment by Dejnaa** [comment]
+1

**Comment by Dejnaa** [comment]
This would be very helpful, and I dont think it requires a lot of coding to make the button also F5?

**Comment by ICCDesi** [comment]
We know that the fact that it doesn't refresh auto its painful. We discussed about it plenty of times in our meetings.

There is a plan for this to make it refreshing in near real time and/or add the button in another topic.

> #31 Update the raid data in nearly real time
